README.md

```diff
@@ -122,25 +122,29 @@ The node has some optional launch parameters that can be specified from the comm
 - `namespace` : namespace that the driver will run in. All services and publishers will be prepended with this, default: `/`
 - `node_name` : name of the driver, default: `microstrain_inertial_driver`
 - `debug`     : output debug logs, default: `false`
-- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](./microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml), default: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
+- `params_file` : path to a parameter file to override the default parameters stored by default in: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
```

Comment:
I would revert this. The purpose of this is to say that the defaults are stored in `params.yml`, and that should be where you look for a list of parameters

Reply:
I've realised that the reason that the `params.yml` file doesn't look like it exists is due to it being in a submodule. When you clone locally and initialise submodules the `params.yml` file appears at that location, but for GitHub markdown it  doesn't liink like that.

Looking at #337 it looks like the fix would be link to the path on the submodule's repo. Could revert and then change the path?